packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,69 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop                      | Type                                                       | Default     | Description                                                                    | Required |
+| ------------------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children                  | `number`                                                   | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent                    | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                       |          |
+| fontWeight                | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                        |          |
+| hasDecimals               | `true`, `false`                                            | `true`      | Indicates whether the amount has or can have decimals                          |          |
+| suffix                    | `true`, `false`                                            | `false`     | Indicates whether a text suffix should be used                                 |          |
+| size                      | `3xlarge`, `2xlarge`, `xlarge`, `large`, `medium`, `small` | `3xlarge`   | The size of the amount to be rendered                                          |          |
+| isSuffixPrefixHighlighted | `true`, `false`                                            | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small |          |
```

Comment:
Where is the prefix?

Reply:
They are referring to the Rupee symbol here.

Reply:
Should we just make this `isHighlighted`, seems too verbose right now and actually we're not highlighting them, instead we're shrinking them 😅

Reply:
That does makes sense and it `isHighlighted` is not too verbose, @kamleshchandnani @chaitanyadeorukhkar @divyanshu013 final call.

Reply:
How about `isAffixHighlighted`?

Prefixes and Suffixes are types of affixes. This way we're not being too vague about what it means too.

Reply:
Sure, this works.